third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -2232,6 +2274,35 @@ resource "google_container_cluster" "with_release_channel" {
 }
 `, clusterName, channel)
 }
+
+func testAccContainerCluster_withTelemetryEnabled(clusterName string, telemetry_type string) string {
+	return fmt.Sprintf(`
+resource "google_container_cluster" "with_cluster_telemetry" {
+  name               = "%s"
+  location           = "us-central1-a"
+  initial_node_count = 1
+
+  cluster_telemetry {
+    type = "%s"
+  }
+}
+`, clusterName, telemetry_type)
+}
+
+func testAccContainerCluster_withTelemetryEnabledForSystemOnlyMetrics(clusterName string, telemetry_type string) string {
+	return fmt.Sprintf(`
+resource "google_container_cluster" "with_cluster_telemetry" {
+  name               = "%s"
+  location           = "us-central1-a"
+  initial_node_count = 1
+  min_master_version = "1.15"
```

Comment:
This is the only difference between this func and the previous one, right? Does it make sense to just have one func and just always set the correct min master version in it?

Reply:
done